src/index.rst

```diff
@@ -3,6 +3,31 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
```

Comment:
If we're adding text that isn't the "toctree" code, we should probably delete this text above, right? Did you build it locally and see if this text appears?

Reply:
That's right, @kathatherine, thanks for pointing it out, I will remove it. And yes I checked it locally. This is how it looks as of now: 
<img width="1440" alt="Screenshot 2022-04-12 at 3 59 00 AM" src="https://user-images.githubusercontent.com/51460047/162843805-224c0c29-d5ea-4f08-9431-7ec58e16eb23.png">

Reply:
Awesome, thanks!